i18n/locales/source/pegasus/mobile.yml

```diff
@@ -1721,3 +1721,8 @@
   page_not_available_title: "Page Not Available"
   page_not_available_description: "We're sorry, the page you requested is no longer available at this address. If you still need access, please email [support@code.org](%{support_email_link})."
   page_not_available_button: "Return home"
+  ! '': 
```

Comment:
not 100% sure what this is, but I expect yaml to do weird stuff so I'm not too worried about it